### DISCUSSION TITLE
Powering the Teensy Breakout Board with external power supply

### DISCUSSION THREAD

**Discussion by hosquiat** [grblHAL/core#254] [Q&A]
Im looking to power the breakout board with an external power supply. I've read through the manual but haven't been able to find this information. Is the only way to power the board to do so while connected via USB?

**Comment by terjeio** [comment]
Which breakout board?

The Teensy itself has to be modified if [connected to an external supply](https://duckduckgo.com/?t=ffab&q=teensy4+external+power&ia=web) it seems.

**Comment by hosquiat** [comment]
Hey thanks for your response. I'm talking about this board here:

https://github.com/phil-barrett/grblHAL-teensy-4.x

**Reply by Avatario34** [reply]
https://www.pjrc.com/store/teensy41.html#power

When powered via Vin the pads need to be cut

**Reply by Avatario34** [reply]
https://github.com/phil-barrett/grblHAL-teensy-4.x/blob/master/T41U5XBB%20v207.pdf

See section "External 5V Feed"

**Reply by hosquiat** [reply]
Thanks for this information. I don't currently have 5V available within the electronics enclosure. To avoid having to wire it up as well as introduce a buck converter, could the Teensy be powered via 12v?  If so, is the process the same?

**Reply by Avatario34** [reply]
I don‘t think so. The T4.1 needs to be powered with 5V. It does have an onboard 3.3V converter but it needs 5V at Vin.
